src/js/form-builder.js

```diff
@@ -65,6 +66,13 @@ function FormBuilder(opts, element, $) {
   formBuilder.colWrapperClass = colWrapperClass
   formBuilder.fieldSelector = opts.enableEnhancedBootstrapGrid ? rowWrapperClassSelector : defaultFieldSelector
 
+  //Initialise HTML sanitizer
+  setSanitizerConfig(opts.sanitizerOptions)
+  if ($(element).closest('form').length) {
+    //Due to Dom Clobbering potential with the stage and the lack of requirement for a Form element, warn for this type of setup
```

Comment:
what is the potential outcome if formBuilder is loaded in a `form` tag?

Reply:
FormBuilder has inputs in the edit panel with name=className and name=name, these clobber form.className form.name.

Reply:
sorry i missed that explanation in chat at first, thanks for repeating here.